### ISSUE TITLE
Large values on the y axis of the space plots are hard to read

### ISSUE THREAD

**Issue by fake-name** [ankane/pghero#214]
![image](https://user-images.githubusercontent.com/1401239/41581304-f78dda90-7352-11e8-9ad1-6dfd0701002a.png)

For reference, yes, this table is 1.59 terabytes. It's also the reason I'd really love https://github.com/ankane/pghero/issues/170.

Basically, I think the plots need to scale to another unit above MB, and ideally should probably chose a unit that has maybe at most 3 zeros in it? I dunno, making nice graphs can be fiddly.

Additionally, the mouseover tool-tip is both kind of fiddly to hit, and unitless:

![image](https://user-images.githubusercontent.com/1401239/41581528-98416c04-7353-11e8-88aa-40f67a805431.png)

What I've found useful for these sort of things (time series graph, single dataset) is to activate the tool-tip for the dataset dependent on the x-position of the mouse *only*. Basically, if the mouse is on the plot, show the highlight that is in the same x-column as the mouse. 

Here's an example from my [Grafana](https://grafana.com/) install:

![mouseover 1](https://user-images.githubusercontent.com/1401239/41581634-f50c4aa8-7353-11e8-87a9-9a4f036f263b.gif)

Basically, the point is you *don't* have to hit the data-point **exactly** to see the relevant value. It makes the plots a LOT easier to use.

As an aside, shipping stats to a statsd metrics aggregator is another interesting idea to consider.

**Comment by ankane** [comment]
Hey @fake-name, thanks for this suggestion as well. Both good ideas. I pushed the tooltip change to master. Will look into adding better display sizes to Chartkick (the charting library) as well.

**Comment by ankane** [comment]
Added better display sizes and cleaned up the tooltip. If you're still using PgHero, please give it a shot and let me know what you think.
